site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Web-to-app and app-to-web measurement'
+subhead: >
+  Learn how your Chrome web app can pass attributions to your Android app.  
+description: >
+  Learn how your Chrome web app can pass attributions to your Android app.    
+date: 2023-08-09
+authors:
+  - maudn
+---
+
+The Privacy Sandbox natively supports cross- web and app attribution, and the Attribution Reporting API allows measurement across mobile browsers and Android apps.
```

Comment:
```suggestion
The Privacy Sandbox natively supports cross-web and app attribution, and the Attribution Reporting API allows measurement across mobile browsers and Android apps.
```

Reply:
Or is that not it? What is it actually trying to say? cross web-and-app?

Reply:
Ya, changing :)